docs/changes/v11.5.0/Server-Changes.adoc

```diff
@@ -14,3 +14,10 @@ A new `pki_ds_url` parameter has been added for `pkispawn` to replace the follow
 A new `pki_http_enable` parameter has been added for `pkispawn`
 to enable/disable the plain HTTP connector in `server.xml`.
 The default value is `True`.
+
+== Remove cert and CSR from _CS.cfg_ ==
+
+The parameters `<subsystem_name>.<cert_id>.cert` and `<subsystem_name>.<cert_id>.certreq` are removed from `CS.cfg` files.
+Certificates are retrieved from the nssdb configured and they are not stored in other places.
+CSR are stored in the folder `<instance_config>/certs` as `<cert_nickname>.csr` and they are retrieved from this location.
+For the cloning operation CSRs have to be specified in `pkispawn` configuration because they cannot be retrieved from the master configuration.
```

Comment:
This means if someone has an automation for the cloning process (e.g. QE, IPA) this change will break it. I think initially we should continue supporting the existing mechanism (i.e. retrieving the CSRs from master) but we can generate a warning saying that we're deprecating this process and they will need to provide the CSRs to pkispawn. Then in the future we can drop the old code.

We probably should also consider providing a tool to export the CSRs more easily, e.g. creating a tarball containing all CSRs so the admin doesn't need to deal with multiple files.

Reply:
You might also want to update the IPA clone test to verify that the `<instance>/certs/<nickname>.csr` files exist on the clones.

Reply:
@edewata I miss this last comment. I'll add before merge or when working on the CSR healtcheck test.